action-types.yml

```diff
@@ -0,0 +1,35 @@
+typingSpec: krzema12/github-actions-typing@v0.3
+inputs:
+  distribution:
+    type: enum
+    allowedValues:
```

Comment:
The standard keys in action and workflow files follow kebab-case,
maybe this file should also use it instead of camelCase?

Reply:
GitHub is not very consistent when it comes to casing - apart form what you wrote, I've seen [snake_case](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_call) as well. But you're right that kebab-case is the most frequent casing, let me change it in the whole typing spec.

Reply:
https://github.com/krzema12/github-actions-typing/issues/10

Reply:
Actually they even have one case I have seen where they used camelCase, for `deprecationMessage`. :-D
But yeah, most is kebab-case, that's what I meant. :-)